native_client/c_speech_features/c_speech_features.h

```diff
@@ -0,0 +1,214 @@
+
+/**
+ * Calculate filterbank features. Provides e.g. fbank and mfcc features for use
+ * in ASR applications.
+ *
+ * Derived from python_speech_features, by James Lyons.
+ * Port by Chris Lord.
+ */
+
+#ifndef __C_SPEECH_FEATURES_H__
+#define __C_SPEECH_FEATURES_H__
+
+#include <math.h>
+
+#define CSF_HZ2MEL(x) (2595 * log10f(1+x/700.0f))
+#define CSF_MEL2HZ(x) (700 * (powf(10.0f, x/2595.0f) - 1))
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
```

Comment:
Can't take any credit, it's just lifted from python_speech_features with the necessary changes so that it applies correctly (hopefully) :)